android-configure

```diff
@@ -23,7 +21,7 @@ except ImportError:
   from distutils.spawn import find_executable as which
 
 print('Node.js android configure: Found Python {}.{}.{}...'.format(*sys.version_info))
-acceptable_pythons = ((3, 11), (3, 10), (3, 9), (3, 8), (3, 7), (3, 6))
+acceptable_pythons = ((3, 11), (3, 10), (3, 9), (3, 8))
```

Comment:
Same.

Reply:
See other response.

Reply:
OK.  I or someone else will revisit these files to complete the task.